### PR TITLE
Add completions for cpupower

### PR DESCRIPTION
**Pull request by llenck**
Add autocompletion for `cpupower`

## TODOs:
- n/a Changes to fish usage are reflected in user documentation/manpages.
- n/a Tests have been added for regressions fixed (n/a)
- [x] User-visible changes noted in CHANGELOG.rst

### CODE REVIEW

File: share/completions/cpupower.fish [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+complete -c cpupower -n "__fish_seen_subcommand_from frequency-set" -s d -l min -x -d "new minimum CPU frequency the governor may select."
+complete -c cpupower -n "__fish_seen_subcommand_from frequency-set" -s u -l max -x -d "new maximum CPU frequency the governor may select."
+complete -c cpupower -n "__fish_seen_subcommand_from frequency-set" -s g -l governor -x -a "(string split ' ' < /sys/devices/system/cpu/cpu0/cpufreq/scaling_available_governors)" -d "new cpufreq governor."
+complete -c cpupower -n "__fish_seen_subcommand_from frequency-set" -s f -l freq -x -d "specific frequency to be set. Requires userspace governor to be available and loaded."
+complete -c cpupower -n "__fish_seen_subcommand_from frequency-set" -s r -l related -d "modify all hardware-related CPUs at the same time"
+
+complete -c cpupower -n "__fish_seen_subcommand_from idle-info" -s f -l silent -d "Only print a summary of all available C-states in the system."
+
+set -l idlestates (cpupower idle-info | awk 'sub(/^Available idle states: /, "")')
```
Comment (faho) [inline share/completions/cpupower.fish:48]: Unfortunately this is translated, so you'll probably need to launch cpupower with `LC_ALL=C cpupower`.

That goes for all of them.

Comment (faho) [reply share/completions/cpupower.fish:48]: Also this is trivially replaced with

```fish
LC_ALL=C cpupower idle-info | string replace -f 'Available idle states: ' ''
```

which avoids the awk dependency.

Comment (llenck) [reply share/completions/cpupower.fish]: Thank you for catching that ;)

**Comment by faho** [comment]
Merged, thanks!

**Pull request merged**